### PR TITLE
Newsletter novembre

### PR DESCRIPTION
**Pull request by Morendil**

### CODE REVIEW

File: content/news/2018-11-14-news.fr.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## International
+
+### Refonte de la documentation
+
+Vous pouvez désormais consulter en un seul endroit le guide de prise en main de la plateforme OpenFisca mais également la documentation de référence de l'API: https://doc.openfisca.org (en anglais)
+
+### Trace: apparition des paramètres
+
+Le endpoint /trace de l'API vous indique désormais quels paramètres législatifs ont été utilisés dans une formule et quelle était leur valeur, afin de vous éviter des aller-retour entre la lecture d'une trace et le legisation explorer.
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
### Trace : apparition des paramètres

Le endpoint `/trace` de l'API Web vous indique désormais quels paramètres législatifs ont été utilisés dans une formule et quelle était leur valeur, afin de vous éviter des aller-retour entre la lecture d'une trace et le legisation explorer.
```

File: content/news/2018-11-14-news.fr.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+[Démo](https://demo.openfisca.org/legislation/swagger#operations-Calculations-trace)
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
[Démo : Testez la nouvelle /trace !](https://demo.openfisca.org/legislation/swagger#operations-Calculations-trace)
```

File: content/news/2018-11-14-news.fr.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+### Recherche usagers
+
+Pour mieux connaitre nos utilisateurs, et guider les évolutions futures, nous avons mené des entretiens, qui se poursuivent. Vous pouvez trouver ici les résultats de la première étape:
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
Pour mieux connaître les utilisateurs OpenFisca, et guider les évolutions futures, Des entretiens sont menés et ce [schéma](https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view) présente les résultats de la première étape.
```

File: content/news/2018-11-14-news.fr.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view
+
+## France
+
+### Fiabilisation des calculs d'impôts
+
+Nous vérifions désormais nos calculs fiscaux en les confrontant aux simulateurs de la DGFIP. Cela nous a permis par exemple de repérer certaines erreurs liées aux règles d'arrondi.
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
Amélioration des calculs fiscaux en les confrontant aux simulateurs de la Direction Générale des Finances Publiques tel que le [simulateur d'impôt 2018](https://www3.impots.gouv.fr/simulateur/calcul_impot/2018/complet/index.htm). [[France #1198](https://github.com/openfisca/openfisca-france/pull/1198)]
```

File: content/news/2018-11-14-news.fr.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## International
+
+### Refonte de la documentation
+
+Vous pouvez désormais consulter en un seul endroit le guide de prise en main de la plateforme OpenFisca et la documentation de référence de l'API Python : https://doc.openfisca.org (en anglais)
+
+### Trace : apparition des paramètres
+
+Le endpoint `/trace` de l'API Web vous indique désormais quels paramètres législatifs ont été utilisés dans une formule et quelle était leur valeur, afin de vous éviter des aller-retour entre la lecture d'une trace et le legisation explorer.
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
Le endpoint `/trace` de l'API Web vous indique désormais quels paramètres législatifs ont été utilisés dans une formule et quelle était leur valeur, afin de vous éviter des aller-retour entre la lecture d'une trace et l'explorateur de la législation. [[Core #765](https://github.com/openfisca/openfisca-core/pull/765)]
```

File: content/news/2018-11-14-news.fr.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+[Démo : Testez la nouvelle /trace !](https://demo.openfisca.org/legislation/swagger#operations-Calculations-trace)
+
+### Recherche usagers
+
+Pour mieux connaître les utilisateurs OpenFisca, et guider les évolutions futures, des entretiens sont menés et ce [schéma](https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view) présente les résultats de la première étape.
+
+https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
```

File: content/news/2018-11-14-news.fr.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+<!--more-->
+
+[_[English version]_](/en/news/2018-11-14-news)
+
+## International
+
+### Refonte de la documentation
+
+Vous pouvez désormais consulter en un seul endroit le guide de prise en main de la plateforme OpenFisca et la documentation de référence de l'API Python : https://doc.openfisca.org (en anglais)
```
Comment (sandcha) [inline content/news/2018-11-14-news.fr.md]:
```suggestion
Vous pouvez désormais consulter en un seul endroit le guide de prise en main de la plateforme OpenFisca et la documentation de référence de l'API Python : [doc.openfisca.org](https://doc.openfisca.org) (en anglais) [[Doc #167](https://github.com/openfisca/openfisca-doc/pull/167)]
```

**Comment by sandcha** [comment]
Thanks @Morendil! 
Needs only to be translated to english and might be adjusted then.

**Comment by sandcha** [comment]
@Morendil Translated in english in last commits.

File: content/news/2018-11-14-news.en.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+## International
+
+### Documentation redesign
+
+You can know check in one site OpenFisca plateform guide and Python API official documentation: [doc.openfisca.org](https://doc.openfisca.org) [[Doc #167](https://github.com/openfisca/openfisca-doc/pull/167)]
+
+### Trace, with parameters!
+
+Web API `/trace` endpoint contains law parameters used by every calculated formula so that, to debug a calculation, you won't need to switch between trace result and legislation explorer anymore. [[Core #765](https://github.com/openfisca/openfisca-core/pull/765)]
```
Comment (Morendil) [inline content/news/2018-11-14-news.en.md]: contains -> now contains… ?

File: content/news/2018-11-14-news.en.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+[Demo: Check new /trace!](https://demo.openfisca.org/legislation/swagger#operations-Calculations-trace)
+
+### Users research
+
+To become better acquainted with OpenFisca users and guide its future evolutions, interviews are conducted. Here is a first [schema](https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view) of the community.  🙂
```
Comment (Morendil) [inline content/news/2018-11-14-news.en.md]: are conducted -> are being conducted
présent progressif dans ce contexte
on peut aussi éviter la voix passive "we are conducting interviews"  ("nous menons des entretiens" aurait ma préférence en français aussi)

File: content/news/2018-11-14-news.en.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+### Users research
+
+To become better acquainted with OpenFisca users and guide its future evolutions, interviews are conducted. Here is a first [schema](https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view) of the community.  🙂 
+
+
+## France
+
+### Taxes calculation improvements
+
+Improved tax calculation thanks to a comparison between OpenFisca results and state ones produced with simulators like [2018 taxes simulator](https://www3.impots.gouv.fr/simulateur/calcul_impot/2018/complet/index.htm). [[France #1198](https://github.com/openfisca/openfisca-france/pull/1198)]
```
Comment (Morendil) [inline content/news/2018-11-14-news.en.md]: Improved -> We improved ?
thanks to a comparison -> by comparing OpenFisca results with the official tax administration simulators, such as…

File: content/news/2018-11-14-news.en.md (hunk 9 above)
Comment (maukoquiroga) [reply content/news/2018-11-14-news.en.md]: We're running :)

**Pull request merged**